.gitignore

```diff
@@ -14,3 +14,5 @@ htmlcov/
 frontend/.nyc_output
 frontend/coverage
 *.log
+backend/vcdat/bin/
```

Comment:
Pretty sure this is a filesystem artefact of a bad build on your system, because I don't have any folder at this path.